apps/events/serializers/event_serializer.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers
+
+from apps.events.models import Event
+from apps.events.models.speaker import Speaker
+from .speaker_serializer import SpeakerSerializer
+
+
+class EventSerializer(serializers.ModelSerializer):
+    speaker = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Event
+        fields = "__all__"
+
+    def get_speaker(self, event):
+        speaker = Speaker.objects.get(id=event.speaker.id)
+        return SpeakerSerializer(speaker).data
+
+
+class CreateEventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Event
+        fields = (
```

Comment:
When fields are these many, we should use exclude instead.
```suggestion
        exclude = (
```